packages/core-mobile/app/contexts/BridgeContext.tsx

```diff
@@ -1,40 +1,12 @@
-import React, {
-  createContext,
-  ReactNode,
-  useCallback,
-  useContext,
-  useEffect,
-  useRef
-} from 'react'
+import React, { ReactNode, useEffect } from 'react'
 import {
   BridgeSDKProvider,
   BridgeTransaction,
-  getMinimumConfirmations,
-  trackBridgeTransaction,
-  TrackerSubscription,
   useBridgeSDK
 } from '@avalabs/core-bridge-sdk'
-import { useDispatch, useSelector } from 'react-redux'
-import { selectActiveAccount } from 'store/account'
-import {
-  addBridgeTransaction,
-  popBridgeTransaction,
-  selectBridgeConfig,
-  selectBridgeTransactions
-} from 'store/bridge'
-import {
-  useAvalancheProvider,
-  useBitcoinProvider,
-  useEthereumProvider
-} from 'hooks/networks/networkProviderHooks'
+import { useSelector } from 'react-redux'
+import { selectBridgeConfig } from 'store/bridge'
```

Comment:
```suggestion
import { selectBridgeConfig } from 'store/bridge/slice'
```
to avoid circular dependency conflict